.changeset/dull-beers-move.md

```diff
@@ -0,0 +1,21 @@
+---
+"astro": minor
+---
+
+Adds an option to suppress warning/errors in adapters' `supportedAstroFeatures`. This is useful when either an warning/error isn't applicable in a specific context or the default one might conflict and confuse users. 
```

Comment:
```suggestion
Adds an option for integration authors to suppress adapter warning/errors in `supportedAstroFeatures`. This is useful when either an warning/error isn't applicable in a specific context or the default one might conflict and confuse users. 
```
I think it's important here to explicitly say that the option is for integration authors, not a user config option to suppress warnings they see in their project because they get annoyed. 😄 

Also, @yanthomasdev , check that I didn't change the nuance, but I think the word order above might be a bit easier to scan.